### DISCUSSION TITLE
some doubts regarding settings

### DISCUSSION THREAD

**Discussion by RaZ0rr-Two** [startup-nvim/startup.nvim#8] [Q&A] [answered]
What is difference between `body` , `body_2` ( [evil.lua](https://github.com/startup-nvim/startup.nvim/blob/master/lua/startup/themes/evil.lua) )and then `section` , `section_2` ( [readme section](https://github.com/startup-nvim/startup.nvim#%EF%B8%8Fthe-basic-structure-of-the-settings) ) ?

**Comment by max397574** [comment, marked as answer]
it doesn't matter how you call your sections

**Reply by RaZ0rr-Two** [reply]
Can you add more clarity? 

Also it might better to give comment like 

`-- Parameters and keys`

Or something similar before color in general settings section.

Shall I give a PR?

**Reply by max397574** [reply]
no you don't have do
I have to rewrite a lot of the docs anyway
What there is atm is just for some people who tested it for me
